.github/workflows/push_pr.yaml

```diff
@@ -158,6 +158,17 @@ jobs:
       - name: Prometheus versions match
         run: make check-prometheus-version
 
+      # TODO Check is failing, for some reason the generated file does not contain
+      # some of the indirect dependencies.
+      # - name: Check THIRD_PARTY_NOTICES.md is up to date
+      #   run: |
+      #     go install go.elastic.co/go-licence-detector@v0.5.0
+      #     make build-license-notice
+      #     if [ ! -z "$(git status --porcelain)" ]; then 
+      #       git diff
+      #       exit 1
+      #     fi
+
```

Comment:
i couldn't make it work 😢  but the idea is there, if we could generate this each time and check if outdated.

Reply:
We could automate it in another PR. No worries.